lib/Repositories/ProjectRepository.php

```diff
@@ -44,7 +44,7 @@ public function findPrimaryProjects(): array
         return $this->findBy([
             'active' => true,
             'integration' => false,
-        ], ['name' => 'asc']);
+        ], ['sortOrder' => 'asc', 'name' => 'asc']);
```

Comment:
Shouldnt that be:

```suggestion
        ], ['sortOrder' => 'desc', 'name' => 'asc']);
```

Reply:
That would move ORM, DBAL and ODM to the end of the project's list in the navigation and homepage. It would be the opposite of what this PR tries to accomplish.